.gitignore

```diff
@@ -5,3 +5,4 @@ tmp
 *.log
 trees*.json
 demo/supercluster.js
+dist/
```

Comment:
Done in https://github.com/mapbox/supercluster/pull/22/commits/81e1b598b6d3176c9f44c38c030b4ef51ea6769c